CHANGELOG.md

```diff
@@ -89,7 +89,8 @@ $indices[$indexName]
     - Elastica\Test\TypeTest::testUpdateDocumentWithIdForwardSlashes
     - Elastica\Test\TypeTest::testUpdateDocumentWithParameter
     - Elastica\Test\TypeTest::testUpdateDocumentWithFieldsSource
-  - Composer installations will no longer include tests and other development files.
+- Composer installations will no longer include tests and other development files.
+- Updated Elastica\Test\Suggest\CompletionTest now payload and output are removed 
```

Comment:
@p365labs Can you rebase on master and move that above the beta1 release line? I'm not even sure we need this changelog entry as it is only an update.